src/bsp/src/stm32/src/interloc/src/InterlocTimeManager.cpp

```diff
@@ -73,3 +73,12 @@ uint64_t InterlocTimeManager::getPollRxStartTs(uint64_t lastSlotStartTs) const {
     return (lastSlotStartTs + m_slotToSlotOffsetDTU - RX_BEFORE_TX_GUARD_US * UUS_TO_DWT_TIME) %
            UINT40_MAX;
 }
+
+uint64_t InterlocTimeManager::getRespRxStartTime(uint64_t pollTxTs, uint8_t responseIdx) {
+    // TODO reference the time with an acutal getPollTxTs
+    // start to listen a little bit before the actual Response is sent.
+    return pollTxTs +
```

Comment:
Could we calculate these values in the update timings and then just return pollTxTs + a precalculated offset?